api/http/handler/endpointgroups/endpointgroup_list.go

```diff
@@ -8,7 +8,18 @@ import (
 	"github.com/portainer/portainer/api/http/security"
 )
 
-// GET request on /api/endpoint_groups
+// @id EndpointGroupList
+// @summary List Endpoint groups
+// @description List all endpoint groups based on the current user authorizations. Will
+// @description return all endpoint groups if using an administrator account otherwise it will
+// @description only return authorized endpoint groups.
+// @description **Access policy**: restricted
```

Comment:
each `@description` tag is a line in the description field. the last line should be in a different line anyway. Since these comments are also visible in code I prefer them to be in separate lines as it makes it easier to view on every screen size

Reply:
I could argue that editor configuration could do a better job fitting comments on every screen size, but I don't think it's a big deal :)
And thanks for clarifying the detail about the last line.